### ISSUE TITLE
Censored Word Replacement

### ISSUE THREAD

**Issue by HeyITGuyFixIt** [MrDogeBro/content_filter#13]
Could you add the ability to replace a matching word with the configured censored word? For example, say you have this line in a JSON file:

```json
{ "find": "find", "word": "word", "censored": "censored" }
```
It would be very useful to have a function that replaced each matching word found with the censored alternative string here or a setting to do this while filtering.

**Comment by MrDogeBro** [comment]
Yeah I can see that being useful. I will work on adding it when I have the time. Thanks.
